contrib/kafka/cmd/controller/main.go

```diff
@@ -68,6 +75,17 @@ func _main() int {
 		return 1
 	}
 
+	TlsEnabled := os.Getenv(kafkaTLSENnabled)
+	if strings.ToLower(TlsEnabled) == "true" {
+		ca := os.Getenv(kafkaCAEnv)
+		provisionerConfig.TlsConfig, err = provisionerController.NewTLSConfig(ca)
+		if err != nil {
+			logger.Error(err, "unable to enable TLS")
+			return 1
+		}
+		logger.Info("successfully enabled TLS", zap.String("secret", kafkaTLSENnabled))
```

Comment:
Why is this marked as the secret, since it's just the string "true" in some capitalization?